test/mounted_files_test.py

```diff
@@ -131,6 +126,17 @@ def test_mounted_files_package(supports_dir, env_mount_files):
     }
 
 
+def test_mounted_files_package_no_automount(supports_dir, env_mount_files, servicer, server_url_env, monkeypatch):
```

Comment:
This tests the new behavior. Previously disabling automount would put the entrypoint file at the root directory instead, regardless how it was being triggered, but now it's being placed at the correct package location (only automounting any __init__.py files on the path to the entrypoint).

This has the advantage that if the user disables automounting and then adds the root package manually, the entrypoint file will not be duplicated in a different location and risk botched imports.